pr-demo.js

```
@@ -3,75 +3,114 @@
 
-// 4. Unclear variable names, inconsistent spacing
-readAndProcess(path.join(__dirname, 'input.json'), function(d){
- let x=processData(d)
-console.log("Processed:",x)
-});
-
-// 5. Unused function with terrible naming
-function doStuff1(){console.log("doing stuff!")}
+// 4. eval usage & global variable leak
+function doEval(str) {
+  eval(str)  // dangerous!
```

Biome (1.9.4)

[error] eval() exposes to security risks and performance issues.

See the MDN web docs for more details.
Refactor the code so that it doesn't need to call eval().

(lint/security/noGlobalEval)